inspectit-oce-bootstrap/src/main/java/rocks/inspectit/oce/bootstrap/instrumentation/ClassLoaderDelegationMarker.java

```diff
@@ -0,0 +1,6 @@
+package rocks.inspectit.oce.bootstrap.instrumentation;
+
+public class ClassLoaderDelegationMarker {
+
+    public static final ThreadLocal<Boolean> CLASS_LOADER_DELEGATION_PERFORMED = new ThreadLocal<>();
```

Comment:
We could use `.... = ThreadLocal.withInitial(() -> false);` here, then we... [next comment]